repository_service_tuf_api/config.py

```diff
@@ -28,14 +34,73 @@ class Config:
         }
 
 
+class PutResponse(BaseModel):
+    data: Dict[str, Any]
+    message: str
+
+    class Config:
+        with open("tests/data_examples/config/update_settings.json") as f:
+            content = f.read()
+        example_settings = json.loads(content)
+
+        schema_extra = {
+            "example": {
+                "message": "Settings successfully updated.",
```

Comment:
The message shows it was already updated and gives the task id.
```suggestion
                "message": "Settings successfully submitted.",
```

Reply:
It is not during the request. It is async. You are sending a task to the message queue with the task.

It will be updated only when the task is processed. It is linked with [this comment](https://github.com/repository-service-tuf/repository-service-tuf-api/pull/387#discussion_r1278291814).